helper/testhelpers/minimal/minimal.go

```diff
@@ -0,0 +1,81 @@
+package minimal
+
+import (
+	"github.com/hashicorp/go-hclog"
+	logicalKv "github.com/hashicorp/vault-plugin-secrets-kv"
+	"github.com/hashicorp/vault/audit"
+	auditFile "github.com/hashicorp/vault/builtin/audit/file"
+	auditSocket "github.com/hashicorp/vault/builtin/audit/socket"
+	auditSyslog "github.com/hashicorp/vault/builtin/audit/syslog"
+	logicalDb "github.com/hashicorp/vault/builtin/logical/database"
+	"github.com/hashicorp/vault/builtin/plugin"
+	"github.com/hashicorp/vault/helper/builtinplugins"
+	"github.com/hashicorp/vault/http"
+	"github.com/hashicorp/vault/sdk/helper/logging"
+	"github.com/hashicorp/vault/sdk/logical"
+	"github.com/hashicorp/vault/sdk/physical/inmem"
+	"github.com/hashicorp/vault/vault"
+	"github.com/mitchellh/copystructure"
+	"github.com/mitchellh/go-testing-interface"
+)
+
+// NewTestSoloCluster is a simpler version of NewTestCluster that only creates
+// single-node clusters.  It is intentionally minimalist, if you need something
+// from vault.TestClusterOptions, use NewTestCluster instead.  It should work fine
+// with a nil config argument.  There is no need to call Start or Cleanup or
```

Comment:
I don't understand why calling `Start` is not needed. Would you please elaborate?

Reply:
There's no need to call Start on any NewTestCluster result these days.  I kept the method around to avoid having to modify a huge number of tests, but it's an empty method as of last year.

Reply:
Nice. I just noticed that my Goland is on 1.12. and it was strange to see this is not needed. Switched to main and see what you mean. Thanks for the context.